tfx/components/trainer/executor.py

```diff
@@ -278,8 +292,12 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
     tf.estimator.train_and_evaluate(training_spec['estimator'],
                                     training_spec['train_spec'],
                                     training_spec['eval_spec'])
-    absl.logging.info('Training complete.  Model written to %s',
-                      fn_args.serving_model_dir)
+
+    # Copy model run information to ModelRun artifact
+    dir_util.copy_tree(fn_args.serving_model_dir, fn_args.log_dir)
```

Comment:
only do it if it's chief worker, otherwise it will be messed up in distributed env